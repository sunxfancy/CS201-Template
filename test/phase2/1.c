
void test() {
  int a, b, c, d, e, f;
  e = b + c;
  if (e > 0) {
    e = a;
  } else {
    e = b + c;
  }
  a = e + c;
}
