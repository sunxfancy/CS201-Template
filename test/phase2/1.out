--test--
entry: a b c
if.then: c e
if.else: c e
if.end: 
